data/data/gcp/bootstrap/main.tf

```diff
@@ -1,3 +1,26 @@
+resource "google_storage_bucket" "ignition" {
+  name = "${var.cluster_id}-ignition"
+}
+
+resource "google_storage_bucket_object" "ignition" {
+  bucket  = "${google_storage_bucket.ignition.name}"
+  name    = "bootstrap.ign"
+  content = var.ignition
+}
+
+resource "google_storage_object_access_control" "ignition-public" {
```

Comment:
this is not acceptable at all. We cannot make the bootstrap.ign contents public.

Reply:
agreed, but is this acceptable to unblock us while we get to a better solution?